### DISCUSSION TITLE
How to ensure the consistency of oap metrics ?

### DISCUSSION THREAD

**Discussion by tjzheng1002** [apache/skywalking#11038] [General]
We already knew merge metrics into cachedMetrics, change only happens inside cachedMetrics.
The PersistenceTimer of different OAP nodes concurrently updates the same metrics to elasticsearch. How can we ensure the consistency of this metrics ?

**Comment by wu-sheng** [comment]
Why that matters? It is impossible for a distributed cluster.
This seems like a strange ask.
